src/h2o_authn/discovery.py

```diff
@@ -56,7 +63,9 @@ def new(
         scope=set_scope,
         expiry_threshold=expiry_threshold,
         expires_in_fallback=expires_in_fallback,
+        http_timeout=http_timeout,
```

Comment:
ultranit: Why not group the `http_` params together?

Reply:
I pass them in the same order as they are defined on `TokenProvider` side.